.pylintrc

```diff
@@ -1,4 +1,4 @@
 [MASTER]
 ignore=tests
 # Disable useless-object-inheritance because we still support py2
-disable=R0205
+disable=R0205,R0801
```

Comment:
I'm not sure if this should be applied globally or just for our modules. @evgeni?

Reply:
I was unable to get the disable to work anywhere else but here.

Reply:
Well, but `pylint` is right, it is duplicated code. So lets unduplicate it :)

https://gist.github.com/evgeni/18e65f302c44a5dd94251d5b6306f71a

Reply:
Does that imply this should be dropped now from the changeset?

Reply:
Err, yeah, of course. Forgot to add that to the patch.